GHermeneus/include/GHermeneus/Instruction.h

```diff
@@ -0,0 +1,95 @@
+//
+// Created by Jelle Spijker on 7/31/20.
+//
+
+#ifndef GCODEHERMENEUS_INSTRUCTION_H
+#define GCODEHERMENEUS_INSTRUCTION_H
+
+#include <string_view>
+#include <utility>
+#include <vector>
+
+#include "GCode.h"
+
+namespace GHermeneus
+{
+    /*!
+     * An dialect interpreted GCode line which consist of a corresponding line number, command key (e.q. G0, M104) and
+     * a vector of Parameters of type double.
+     *
+     * @brief An interpreted GCode function
+     * @tparam SSV_T the State Space Vector type
+     * @tparam T the magnitude type
+     */
+    template<typename SSV_T, typename T>
+    struct Instruction
+    {
+        Instruction() : line_no(-1), cmd("") {}; // Todo figure out why I need this constructor
```

Comment:
There's probably a hidden way you construct these where you don't actually want them (before you copy the one that you actually meant to construct in its' place). I'll have a look and see if I can spot the place in this code.

Reply:
Thx, It wasn't  evidently obvious on first eye for me and I didn't spend that much investigation on it. So I chose to leave it be for now. First make it work then optimize.